wapopartners/arctrack
# Begin removing 'store' utility & updating trackJs API

## On this branch
_Remove `Store` from `trackJs`_ 
_Remove class reference (`target`) from `trackJs` callback parameters_
_Pass along class instance (`this`) as `trackJs` callback parameter_

## To test
* make sure you have the most up-to-date version of `yarn`
* `yarn && yarn test`
* make sure the tests pass

## QA 
**NA**
